bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -768,7 +799,51 @@ private int updateFromSchema3toSchema4(int schemaVersionNumber, CaseDbConnection
 			closeResultSet(resultSet);
 			closeStatement(statement);
 		}
+	}
 
+	/**
+	 * Updates a schema version 4 database to a schema version 5 database.
+	 *
+	 * @param schemaVersionNumber The current schema version number of the
+	 *                            database.
+	 * @param connection          A connection to the case database.
+	 *
+	 * @return The new database schema version.
+	 *
+	 * @throws SQLException     If there is an error completing a database
+	 *                          operation.
+	 * @throws TskCoreException If there is an error completing a database
+	 *                          operation via another SleuthkitCase method.
+	 */
+	private int updateFromSchema4toSchema5(int schemaVersionNumber, CaseDbConnection connection) throws SQLException, TskCoreException {
+		if (schemaVersionNumber != 4) {
+			return schemaVersionNumber;
+		}
+
+		Statement statement = null;
+		try {
+			statement = connection.createStatement();
+			// Add the review_statuses lookup table.
+			if (this.dbType.equals(DbType.SQLITE)) {
+				statement.execute("CREATE TABLE review_statuses (review_status_id INTEGER PRIMARY KEY, review_status_name TEXT NOT NULL, display_name TEXT NOT NULL)");
+			} else {
+				statement.execute("CREATE TABLE review_statuses (review_status_id BIGSERIAL PRIMARY KEY, review_status_name TEXT NOT NULL, display_name TEXT NOT NULL)");
+			}
+
```

Comment:
I think it can be INTEGER in both places, since we are supplying it instead of doing an auto-increment.